CHANGELOG_PENDING.md

```diff
@@ -1,4 +1,4 @@
-## v0.33.7
+## v0.34
```

Comment:
Possibly in the future we add `Unreleased` so we don't have to update this on releases?